WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -126,10 +126,14 @@ import Combine
     // https://github.com/woocommerce/woocommerce-ios/issues/13207
     let orderStageSubject = PassthroughSubject<PointOfSaleDashboardViewModel.OrderStage, Never>()
     let orderStagePublisher = orderStageSubject.eraseToAnyPublisher()
+    let totalsViewModel = TotalsViewModel(orderService: POSOrderPreviewService(),
+                                              cardPresentPaymentService: CardPresentPaymentPreviewService(),
+                                              currencyFormatter: .init(currencySettings: .init()))
```

Comment:
While this happens on the `#Preview`, declaring a new instance of `TotalsViewModel` in order to inject it into the VM for the `CartView` seems to indicate that the design of the dependency graph is not quite right. I'd say the `CartView` should know nothing about the Totals VM and perhaps this shouldn't be injected here.

Reply:
Addressed in commit 35370c72d7b0f5f967bfaa9304c256416372c12f.